pages/blog/miniseries-part_1.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Mini series - Speed up your development process (part 1)"
+date: 2021-04-28T06:00:00+01:00
+type: Engineering
+tags:
+  - tools
+  - code generation
+cover: /img/posts/jonaslagoni-miniseries-part1/blog-miniseries-cover.webp
+authors:
+  - name: Jonas Lagoni
+    photo: /img/avatars/jonaslagoni.webp
+    link: https://github.com/jonaslagoni
+    byline: AsyncAPI Maintainer
+featured: true
+---
+
+How can you utilize code generation to speed up the development process and only focus on what is important, the business logic? In this mini-series we will explore the ways AsyncAPI and code generation can work hand in hand beyond generating documentation. 
+
+Structure of the miniseries:
+* **Part 1: Designing the API's with AsyncAPI**
+* Part 2: Implementing the applications using code generation
+* Part 3: Black-box testing the applications using code generation
+* Part 4: Introducing new changes when using code generation
+* Part 5: The path to 1 billion players - Scaling the applications and finding bottlenecks with code generation
+
+**Don't see this blog post series as anything other then an example workflow, this is purely how I do it with my applications and how I use AsyncAPI and it's tooling to my advantage. Use this as an inspiration to finding an approach that works for you.**
```

Comment:
```suggestion
> Don't see this blog post series as anything other then an example workflow, this is purely how I do it with my applications and how I use AsyncAPI and it's tooling to my advantage. Use this as an inspiration to finding an approach that works for you.
```

better do it as a kind of note as such a big block of text that is bold is not nice to read